typescript/cosmos-sdk/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@hyperlane-xyz/cosmos-sdk",
+  "version": "1.0.0",
+  "description": "Hyperlane TypeScript SDK for the Cosmos Hyperlane SDK module",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "types": "dist/index.d.ts",
+  "files": [
+    "/dist"
+  ],
+  "homepage": "https://www.hyperlane.xyz",
+  "repository": "https://github.com/hyperlane-xyz/hyperlane-monorepo",
+  "keywords": [
+    "hyperlane",
+    "cosmos",
+    "blockchain",
+    "sdk"
+  ],
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "rimraf dist && tsc",
```

Comment:
Would it be possible to add the types building step to the `build` script and add the types to the `.gitignore`? Considering that the types folder contains autogenerated code I think it would be better to just build it when running `yarn build` instead of committing it with the non-generated files, this would also help in reducing the pr size

Reply:
It would be possible, however the types are currently generated with docker and this is for a good reason since the user needs to have the proto buf cli installed for the proto generation (where the output can differ from version and architecture). So either docker must be active during the build step when building the monorepo or the user has to install the correct buf cli when we want to generate the types before the build step.

Reply:
We can add docker to the required tools for working with the monorepo and update the README so that people setting it up know that they need to have it active, wdyt?